go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/kubevirt/kubevirt-tekton-tasks
 
 go 1.22.7
+toolchain go1.23.3
```

Comment:
Can we bump the go dependency instead?

Reply:
No we can't use this new go version, because we would break the d/s builds